ideascube/catalog_utils/package.py

```diff
@@ -0,0 +1,55 @@
+
+
+import yaml
+import os.path as op
+import zipfile
+import hashlib
+import codecs
+import csv
+
+
+class MediaCenterPackage:
```

Comment:
We agreed not to merge these, at least for now, since this code is probably going to evolve quite a bit.